docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "ArraySliceDynamicallyEncodedBaseType",
+        "summary": "Index range accesses on arrays with dynamically encoded base types can result in invalid array accesses.",
+        "description": "For arrays with dynamically sized base types, index range accesses that use a start expression that is non-zero will result in invalid array slices. Any index access to such array slices will result in data being read from incorrect calldata offsets.",
```

Comment:
This does sound rather harsh - should I justify it still being *very low* by explaining that it's scarcely implemented and unlikely being used so far?

Reply:
Severity is a combination of:
 - likelihood of being used
 - detectability in tests
 - potential for abuse

Reply:
Yeah - I'd say it's not that likely to be used, since such types are probably not that common in calldata, there's not much one can do with array  slices yet (although they *can* be converted back to memory and passed on to other functions or ABI-encoded and that *would* be subject to this bug), and it should be immediately noticable by tests.

So I think "very low" is fine, but should I elaborate in the description a bit more? I already extended it a bit now.